css/styles.css

```diff
@@ -1,535 +1,1148 @@
-  vertical-align: middle;
-  background-color: #fff;
-  color: #396df2;
-  border: solid 1px #396df2;
-  padding: 12px;
-  border-radius: 4px;
-  margin-left: 5%;
-  margin-right: auto;
-}
+  .see-project:hover {
+    font-size: 17px;
+    line-height: 24px;
+    font-weight: 500;
+    text-align: center;
+    vertical-align: middle;
+    background-color: #6070ff;
+    color: #fff;
+    border: solid 1px #fff;
+    padding: 12px;
+    border-radius: 4px;
+  }
 
-.my-resume:hover {
-  font-size: 17px;
-  line-height: 24px;
-  font-weight: 500;
-  text-align: center;
-  vertical-align: middle;
-  background-color: #6070ff;
-  color: #fff;
-  border: solid 1px #fff;
-  padding: 12px;
-  border-radius: 4px;
-  margin-left: 5%;
-  margin-right: auto;
-}
+  .work1-img {
+    display: block;
+    width: 90%;
+    height: auto;
+    margin-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
+  }
 
-.languages {
-  display: flex;
-  justify-content: space-between;
-  margin: 15px auto 15px 26px;
-  width: 80%;
-}
+  /* Section About Myself */
+  .about_myself {
+    width: 80%;
+    margin-bottom: 120px;
+    color: #172b4d;
+    display: flex;
+    padding: 0% 10% 0% 10%;
+  }
 
-.frameworks,
-.skills {
-  display: flex;
-  justify-content: space-between;
-  margin: 15px auto 30px 26px;
-  width: 80%;
-}
+  .first {
+    width: 45%;
+    padding-right: 15px;
+  }
 
-.lang1 {
-  width: 85%;
-  margin: 10px auto 10px auto;
-  display: flex;
-  flex-direction: column;
-}
+  .about {
+    width: 100%;
+    color: #091e42;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+  }
 
-.row1 {
-  width: 100%;
-  display: flex;
-  padding-top: 4px;
-  padding-bottom: 3px;
-  background-color: #f7f7f9;
-}
+  .desc4 {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    color: #344563;
+    margin-bottom: 28px;
+  }
 
-.li-sp {
-  margin-bottom: 12px;
-}
+  .container-4 {
+    display: flex;
+  }
 
-.arrow-down {
-  width: 24px;
-  height: 24px;
-  background-color: #fff;
-  background-blend-mode: multiply;
-  vertical-align: middle;
-}
+  .my-resume {
+    font-size: 17px;
+    line-height: 24px;
+    font-weight: 500;
+    text-align: center;
+    vertical-align: middle;
+    background-color: #fff;
+    color: #396df2;
+    border: solid 1px #396df2;
+    padding: 12px;
+    border-radius: 4px;
+  }
 
-.arrow-right {
-  width: 24px;
-  height: 24px;
-  background-color: #fff;
-  background-blend-mode: multiply;
-  vertical-align: middle;
-}
+  .my-resume:hover {
+    font-size: 17px;
+    line-height: 24px;
+    font-weight: 500;
+    text-align: center;
+    vertical-align: middle;
+    background-color: #6070ff;
+    color: #fff;
+    border: solid 1px #fff;
+    padding: 12px;
+    border-radius: 4px;
+  }
 
-.lang,
-.frame,
-.skill {
-  font-size: 20px;
-  font-weight: 500;
-  line-height: 24px;
-}
+  .second {
+    width: 40%;
+    display: flex;
+    flex-direction: column;
+    padding-left: 15px;
+  }
```

Comment:
- [ ] Dear @jcmunav63, For the `about me` section. the expected behavior is for the left one to take 40% and the right one to the remaining 60% of the page's width. this makes your design looks like the one in figma. 
- [ ] Please also consider adding the lines under each  `Language`, `Framework`, and `Skill`. This is to make your design the exact clone of the given one.
### Figma design 
![image](https://user-images.githubusercontent.com/69077061/230749545-1a29203e-eb6f-4add-b940-b95ffe176188.png)

### Your design 
![image](https://user-images.githubusercontent.com/69077061/230749555-872878ed-d765-4e2a-9c37-7cc225610d28.png)